pkgs/thunderbird-extensions/default.nix

```diff
@@ -0,0 +1,16 @@
+{ callPackage, thunderbird-utils, overrides ? {}, ... }:
+
+{
+  enigmail = thunderbird-utils.buildMozillaExtension {
+    pname = "enigmail";
+    version = "2.1.6";
+
+    id = "71";
```

Comment:
Unused?

Reply:
Couldn't get the Makefile in the repository to build correctly (if I remember correctly) so this is directly fetched from mozilla, which has a component in the path (not quite sure if it's the addon id but that's what I called it).